### PR TITLE
Better snapshot readability

### PR DESCRIPTION
**Pull request by Kayanski**
This PR aims to add better readibility to snapshots taken using the take_storage_snapshot macro.

This adds : 
- Print the contract id instead of the contract address in the snapshot name
- Parse the storage to readable strings instead of array of bytes

Closes ORC-54 and ORC-55.

### CODE REVIEW

File: cw-orch/src/snapshots/cw_orch__snapshots__tests__snapshot_test-counter_contract.snap [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+source: cw-orch/src/snapshots.rs
+expression: "::cw_orch::snapshots::parse_storage(chain.app.borrow().dump_wasm_raw(&contract_addr))"
+---
+- - contract_info
+  - "{\"contract\":\"crates.io:counter\",\"version\":\"0.11.0\"}"
+- - state
+  - "{\"count\":1,\"owner\":\"sender\"}"
```
Comment (Buckram123) [inline cw-orch/src/snapshots/cw_orch__snapshots__tests__snapshot_test-counter_contract.snap:8]: This output and filename is much clearer!

File: cw-orch/src/snapshots.rs [hunk 2]
```diff
@@ -1,30 +1,41 @@
 //! Defined the snapshot testing macro on the Mock environment
 //! This is included here and not in the mock package because it needs to import traits to work
 
+/// Function helper used to parse storage into readable strings
+pub fn parse_storage(storage: Vec<(Vec<u8>, Vec<u8>)>) -> Vec<(String, String)> {
```
Comment (Buckram123) [inline cw-orch/src/snapshots.rs:5]: nit: this function doesn't benefit from ownership of the storage

File: cw-orch/src/snapshots.rs [hunk 3]
```diff
@@ -1,30 +1,41 @@
...
+
 #[macro_export]
 macro_rules! take_storage_snapshot {
     ($chain: ident, $name: literal) => {
         // We register and test a snapshot for all contracts storage
         use ::cw_orch::environment::{ChainState as _, StateInterface as _};
         let all_contract_addresses = $chain.state().get_all_addresses()?;
-        for (_id, contract_addr) in all_contract_addresses {
+        for (id, contract_addr) in all_contract_addresses {
             ::cw_orch::insta::assert_yaml_snapshot!(
```
Comment (Buckram123) [inline cw-orch/src/snapshots.rs:33]: Is it possible to have a single "assert snapshot" after loop? Right now I have to re-run snapshot test for every contract address

**Review by Buckram123 — APPROVED** [review]
Nice to see snapshots being more human-readable! :rocket:

File: cw-orch/src/snapshots.rs (hunk 2 above)
Comment (Kayanski) [reply cw-orch/src/snapshots.rs]: Corrected

**Pull request merged**